src/main/java/uk/gov/digital/ho/hocs/casework/domain/model/StageWithCaseData.java

```diff
@@ -74,9 +82,10 @@ public class StageWithCaseData extends BaseStage {
 
     @Getter
     @Setter
-    @Transient
-    @JsonInclude(JsonInclude.Include.NON_NULL)
-    private ArrayList<String> tag;
+    @OneToMany(fetch = FetchType.EAGER)
+    @JoinColumn(name = "case_uuid", referencedColumnName = "case_uuid")
+    @Where(clause = "deleted_on IS NULL")
```

Comment:
This is a bit of a pain - if this was a native query it would be simpler. I think this is okay though...